### ISSUE TITLE
Improve UI/UIX of CheckBox Toggle columns

### ISSUE THREAD

**Issue by georgegevoian** [gristlabs/grist-core#1824]
### Describe the problem to be solved

After the recent improvements to the appearance of Switch columns, some of the issues with CheckBox columns have become more apparent, like the lack of visibility in the unchecked state, and the overall look and feel not quite being up to our standards.

We briefly considered making Switch the default for Toggle columns, but that doesn't directly address the problem to be solved.

### Describe the solution you would like

CheckBox columns should be more usable and accessible, and have a nicer looking design that matches the general style of Switch columns.

**Comment by Gwabix** [comment]
I fully agree with the point made regarding the appearance and visibility of unchecked checkboxes. I've submitted a pull request to address this (https://github.com/gristlabs/grist-core/pull/2178).
I hope to see this solution (or another one) implemented in Grist soon :)
